packages/gatsby-theme-carbon/src/styles/internal/resources.scss

```diff
@@ -13,5 +13,8 @@
 @use '@carbon/react/scss/type' as *;
 // $standard-easing $transition-x
 @use '@carbon/react/scss/motion' as *;
+// @include visually-hidden;
+@use '@carbon/react/scss/utilities/visually-hidden' as *;
```

Comment:
Should it be importing from `/styles` instead of `/react`?

Reply:
@sstrubberg I stand corrected. In this case the visually-hidden helper isn't provided through `@carbon/react` yet. I'll fix this to use the hidden styling directly instead of having to rely on `@carbon/styles`.